### ISSUE TITLE
Change capture format to asciicast v2

### ISSUE THREAD

**Issue by toxyl** [toxyl/ossh#9]
see https://github.com/asciinema/asciinema/blob/master/doc/asciicast-v2.md

example:
```json
{"version": 2, "width": 80, "height": 24, "timestamp": 1504467315, "title": "Demo", "env": {"TERM": "xterm-256color", "SHELL": "/bin/zsh"}}
[0.248848, "o", "\u001b[1;31mHello \u001b[32mWorld!\u001b[0m\n"]
[1.001376, "o", "That was ok\rThis is better."]
[2.143733, "o", " "]
[6.541828, "o", "Bye!"]
```

**Comment by toxyl** [comment]
use https://github.com/asciinema/asciinema-player for playback

**Comment by toxyl** [comment]
Implemented in PR #10
